portugues/colabora/semdiferencas/pentefino/pentefino.md

```diff
@@ -2,4 +2,4 @@ Depois de passar horas analisando e depurando, você
 
 [Descobre um erro de lógica](../../comdiferencas/diff/diff.md)
 
-[Já não consegue raciocinar e fica babando no teclado de tanto sono](../../colaborar.md)
```

Comment:
I appreciate it @cbuckey-uda! I hope people like it.
Btw, I loved the course, and let my thanks to you all on the discuss forum. Cheers!